server/routes/pinIPFS.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction } from 'express';
+import { DB, sequelize } from '../database';
+import ipfs from '../util/ipfs';
+export const Errors = {
+  NotLoggedIn: 'Not logged in',
+  InvalidAddress: 'Invalid address',
+  NoBlobPresent: 'No JSON blob was input',
+};
+
+const pinIPFS = async (
```

Comment:
Ideally we should use the newer typescript + error logic for any new routes, like in `server/routes/viewOffchainVote.ts`, but I wasn't clear about that + this is fine.

Reply:
Changed this

Reply:
Doesn't look changed but ok.